src/osip-model/src/main/java/edu/kit/pse/osip/core/model/base/Pipe.java

```diff
@@ -28,7 +29,26 @@ public Pipe(float crosssection, int length) {
         }
         this.crosssection = crosssection;
         this.length = length;
+        defaultThreshold = (byte) 100;
     }
+
+    /**
+     * Construct a Pipe.
+     * @param crosssection Crosssection of the pipe in cm².
+     * @param length Length in cm.
+     * @param defaultThreshold The default threshold
+     * @throws IllegalArgumentException if crosssection or length are negative or zero.
+     */
+    public Pipe(float crosssection, int length, byte defaultThreshold) {
```

Comment:
Normal mag ich solche Änderungen ja nicht, aber in dem Fall wäre es sonst zu inkonsistent